cumulusci.yml

```diff
@@ -290,18 +299,26 @@ flows:
                 options:
                     unmanaged: False
 
-    install_regression:
+    config_regression:
         steps:
-            # Add QA metadata needed for regression
-            5:
+            1:
                 task: deploy_qa_config
                 options:
                     unmanaged: False
-            6:
+            2:
                 task: update_admin_profile
                 options:
                     managed: true
 
+    config_trial:
+        steps:
+            1:
+                flow: config_qa
+            2:
+                task: enable_customizable_rollups
+            3:
+                task: enable_incremental_rollups
```

Comment:
We could arrange for these to happen with a single anonymous Apex call instead of 2, but it's not a big deal.